public/locations-list-map.js

```diff
@@ -167,11 +167,12 @@ function toHtmlSnippets(data_by_location, filters) {
     for (const city of Object.keys(cities).sort()) {
       const entryLines = [];
       for (const entry of cities[city]) {
-        const name = entry["What is the name of the hospital or clinic?"];
-        const address = entry["Street address for dropoffs?"];
-        const instructions = entry["Drop off instructions, eg curbside procedure or mailing address ATTN: instructions:"];
-        const accepting = entry["What are they accepting?"];
-        const will_they_accept = entry["Will they accept open boxes/bags?"];
+
+        const name = entry[$.i18n("ftm-sheet-hospital-name")];
```

Comment:
I believe these five strings are actually embedded in the header row of google sheet that underlies the site, and I'm not sure if they should be translated - flagging for @graemer to take a look.

Reply:
Yes. don't touch these right now. I'm going to fix them so they're not localized in a bit.

Reply:
I did this intentionally, but on second thought I'll move them to a separate object so that their purpose is more clear.

Reply:
Following up: I added strings for these because they will be translated for anyone who translates the form. But I agree that just translating them like other locale strings is a mistake, since they need to match the sheet exactly. The comment I had in there was clearly not enough to call them out because y'all didn't see it. :)

I moved them into their own separate object.